tests/test_app.py

```diff
@@ -89,5 +88,4 @@ def test_app_run(mocked_asyncio):
     mocked_event_loop = mocked_asyncio.get_event_loop.return_value
 
     mocked_asyncio.get_event_loop.assert_called_with()
-    mocked_event_loop.create_task.assert_called_with('fake')
```

Comment:
same here :)

Reply:
I'm unsure if I understand this one. I remove it because tasks are an empty list and this create be called in an iterator